api/v1alpha1/suite_test.go

```diff
@@ -47,28 +42,8 @@ func TestAPIs(t *testing.T) {
 
 var _ = BeforeSuite(func(done Done) {
 	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
-
-	By("bootstrapping test environment")
-	testEnv = &envtest.Environment{
```

Comment:
This test have been "removed" because it was failling on CI (dependency to kube-builder and etcd binary). Right now kube-builed is installed on CI so it should work.
So :
- remove it completely and plan in follow up e2e test
- or restore it

Reply:
removed, e2e to follow